portfolio/src/main/webapp/index.html

```diff
@@ -8,21 +8,28 @@
   </head>
   <body>
     <div id="pageWrapper">
+      <div id="navbar">
+        <div class="navButton" onClick="highlightSection('basicInfo')">Introduction</div> 
+        <div class="navButton" onClick="highlightSection('hobbyWrapper')">Hobbies</div> 
+        <div class="navButton" onClick="highlightSection('projects')">Projects</div> 
+      </div> 
+
       <div id="introduction">
         <h1>Hi there!</h1>
         <div id="headshotWrapper">
-        <img id="headshot" src="city.jpg" alt="city skyline"/> 
+          <img class="headshot" id="image" src="city.jpg" alt="skyline of Beijing"/>
           <div id="caption">My name is Holly, and this is a small place on the Internet that has my portfolio.</div> 
```

Comment:
I'd suggest that you change the ID to something like "headshotCaption", because there can be other captions in the same page.

Reply:
Done.